openhands/agenthub/codeact_agent/function_calling.py

```diff
@@ -192,7 +194,7 @@ def __init__(self):
                     'type': 'string',
                     'description': 'The absolute path to the file to be edited.',
                 },
-                'new_content_draft': {
+                'content': {
```

Comment:
@xingyaoww this should be the correct value right? I remember someone mentioned it in slack